kruize-clowdapp.yaml

```diff
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${KRUIZE_CPU_LIMIT}
             memory: ${KRUIZE_MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /health
+            port: ${AUTOTUNE_SERVER_PORT}
```

Comment:
```suggestion
            port: 10000
```
clowder operator does not support parameterize port value. So operator will not parse `${AUTOTUNE_SERVER_PORT}` to `10000`. When tried to run in EE got the below error - 
```
error creating resource Deployment kruize-recommendations: Deployment.apps "kruize-recommendations" is invalid: [spec.template.spec.containers[0].livenessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must be no more than 15 characters, spec.template.spec.containers[0].livenessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must contain only alpha-numeric characters (a-z, 0-9), and hyphens (-), spec.template.spec.containers[0].livenessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must contain at least one letter (a-z), spec.template.spec.containers[0].readinessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must be no more than 15 characters, spec.template.spec.containers[0].readinessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must contain only alpha-numeric characters (a-z, 0-9), and hyphens (-), spec.template.spec.containers[0].readinessProbe.httpGet.port: Invalid value: "${{AUTOTUNE_SERVER_PORT}}": must contain at least one letter (a-z)]
```
Please hardcode the port in probes.